pkg/asset/agent/manifests/clusterimageset.go

```diff
@@ -57,7 +57,7 @@ func (a *ClusterImageSet) Generate(ctx context.Context, dependencies asset.Paren
 	dependencies.Get(releaseImage, installConfig, agentWorkflow, clusterInfo)
 
 	switch agentWorkflow.Workflow {
-	case workflow.AgentWorkflowTypeInstall, workflow.AgentWorkflowTypeInstallInteractiveDisconnected:
+	case workflow.AgentWorkflowTypeInstall:
```

Comment:
for OVE, will the default workflow be still set as Install? Or it will be unset and have an empty value